NSS-Day-Cohort-42/rare-media-moguls
# Tags List

#### Changes Made
1. Added files `TagsList.js` and `Tag.js` to `tags` directory.
1. Modified `ApplicationViews.js` to include path routed to `TagsList` view.
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout sm-tagsList
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
>From Nav bar click `Tags Management`
> On this page, you should be able to view a list of tags including and edit and delete button for each (functionality for these will come later)
>You should also be able to view an `Add New Tag` Button that will render a form later to add a tag. 
>Make sure that the path ends in `/tags`
